### DISCUSSION TITLE
Grey Mystic

### DISCUSSION THREAD

**Discussion by TomekPL-1** [secretcowlevel/DoomtrooperPublic#45] [Card Balance]
Currently he is:
Grey Mystic (MISHIMA) [C]
While in Cover, generate 1 [DP] per turn.

F: 2 S: 1 A: 2 V: 3

I propose:
Grey Mystic (MISHIMA) [C]
While in Cover, Grey Mystic provide you with either one non attack action or 3 [DP] per turn. You may take an action or D during the turn you muster him.

F: 2 S: 1 A: 2 V: 3

I realise that it is strong ability, but on the other hand he is very easy to kill, he ocuppy 1 warrior slot, to use his ability you need to spend 2 actions - muster and cover. And if he is only warrior in SQUAD he cannot go in to cover.

**Comment by 21-ric** [comment]
This WARRIOR need to be update. 🙂

1 suggestion based on digital card game :

Grey Mystic (MISHIMA) [C]
While in Cover, generate 1 [DP] per turn.
May enter cover at any time without performing a Seek Cover Action.
F: 2 S: 1 A: 2 V: 3

1 suggestion based on original card game :

Grey Mystic (MISHIMA) [C]
May be given Ki Powers during your turn with no action cost.
F: 2 S: 1 A: 2 V: 3

![image](https://user-images.githubusercontent.com/83040170/119909428-188dbe80-bf55-11eb-8e34-5d5bbb7f53bd.png)

**Reply by Sol-au-fein** [reply]
Wow, they were good warriors in original

**Reply by TomekPL-1** [reply]
Sol-au-fein I never seen Grey Mystic in use in table top doomtrooper. He was too weak to have him in deck.

**Comment by david-roxygen** [comment]
I agree with the following version

Grey Mystic (MISHIMA) [C]
May be given Ki Powers during your turn with no action cost.
F: 2 S: 1 A: 2 V: 3

**Comment by TomekPL-1** [comment]
Grey Mystic (MISHIMA) [C]
While in play generate 1 [DP] on the beginning of your draw step. While in Cover, increase you maximum hand size +1.
